src/FondOfSpryker/Glue/ConditionalAvailabilityPageSearchRestApi/Plugin/ConditionalAvailabilityPageSearchRestApiExtension/QuantityRestConditionalAvailabilityPeriodMapperPlugin.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace FondOfSpryker\Glue\ConditionalAvailabilityPageSearchRestApi\Plugin\ConditionalAvailabilityPageSearchRestApiExtension;
+
+use FondOfSpryker\Glue\ConditionalAvailabilityPageSearchRestApiExtension\Dependency\Plugin\RestConditionalAvailabilityPeriodMapperPluginInterface;
+use Generated\Shared\Transfer\RestConditionalAvailabilityPeriodTransfer;
+use Spryker\Glue\Kernel\AbstractFactory;
+
+class QuantityRestConditionalAvailabilityPeriodMapperPlugin extends AbstractFactory implements RestConditionalAvailabilityPeriodMapperPluginInterface
+{
+    protected const PERIOD_DATA_KEY_QUANTITY = 'quantity';
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param array $periodData
+     * @param \Generated\Shared\Transfer\RestConditionalAvailabilityPeriodTransfer $restConditionalAvailabilityPeriodTransfer
+     *
+     * @return \Generated\Shared\Transfer\RestConditionalAvailabilityPeriodTransfer
+     */
+    public function mapPeriodDataToRestConditionalAvailabilityPeriodTransfer(
+        array $periodData,
+        RestConditionalAvailabilityPeriodTransfer $restConditionalAvailabilityPeriodTransfer
+    ): RestConditionalAvailabilityPeriodTransfer {
+        if (!isset($periodData[static::PERIOD_DATA_KEY_QUANTITY])) {
+            return $restConditionalAvailabilityPeriodTransfer;
+        }
+
+        return $restConditionalAvailabilityPeriodTransfer->setQty($periodData['quantity']);
```

Comment:
Why not use static::PERIOD_DATA_KEY_QUANTITY? :)

Reply:
Thx @julianzimmermann. Good point.